src/firefly/js/charts/ui/PlotlyWrapper.jsx

```diff
@@ -280,9 +280,10 @@ export class PlotlyWrapper extends Component {
                         this.restyle(this.div, Plotly, data, dataUpdate, dataUpdateTraces);
                         break;
                     case RenderType.RELAYOUT:
-                        Plotly.relayout(this.div, layoutUpdate);
+                        Plotly.relayout(this.div, layout);
                         break;
                     case RenderType.RESTYLE_AND_RELAYOUT:
+                        //ToDo: Remove this? this is probably never called, since when RenderType is RESTYLE_AND_RELAYOUT, we set it to NEW_PLOT anyway
                         this.restyle(this.div, Plotly, data, dataUpdate, dataUpdateTraces);
```

Comment:
No need for this comment.  It's probably not accurate.  This is only true when `useScatterGL` is false.